docs/index.rst

```diff
@@ -9,6 +9,7 @@ vehicle. Below are the five main modules of the system.
    :maxdepth: 2
 
    bluetoothctl
+   camera
```

Comment:
Right above this there's a sentence that says "five main modules." Can you update that to six